src/mongocxx/collection.hpp

```diff
@@ -350,6 +350,10 @@ class MONGOCXX_API collection {
     /// @return The original or replaced document.
     /// @throws exception::write if the operation fails.
     ///
+    /// @note
```

Comment:
Right now, this is the only thing possible with the C driver.
We had a bit of discussion [on CDRIVER-945](https://jira.mongodb.org/browse/CDRIVER-945) about this.